requirements.txt

```diff
@@ -7,3 +7,4 @@ PySide6_Addons>=6.5.1.1,<=6.6.3.1
 PySide6_Essentials>=6.5.1.1,<=6.6.3.1
 typing_extensions>=3.10.0.0,<=4.11.0
 ujson>=5.8.0,<=5.9.0
+parametrized==0.9.0
```

Comment:
```suggestion
parameterized==0.9.0
```

Reply:
Thank you for the feedback! I have added a requirements-dev.txt as discussed in this MR with pytest.